modules/caffeCoder/src/CaffeFeatExtractor.hpp

```diff
@@ -19,17 +19,17 @@
 #include "caffe-version.h"
 
 #if (CAFFE_MAJOR >= 1)
-	// new caffe headers
-	#include "caffe/caffe.hpp"
- 	#include "caffe/layers/memory_data_layer.hpp"
+    // new caffe headers
+    #include "caffe/caffe.hpp"
+    #include "caffe/layers/memory_data_layer.hpp"
 #else
```

Comment:
True: I think we can leave it as it is the same kind of check that we will need to do in the near future on the different versions, but we can modify it for the moment in order to just require version higher than 1.